content/400-pulse/600-faq.mdx

```diff
@@ -44,5 +44,26 @@ Pulse’s database compatibility list will be different from Prisma Client's dat
 | -------- | -------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Railway  | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Railway.  |
 | Supabase | ✅       | Requires [`sslmode=disable`](/orm/overview/databases/postgresql#configuring-an-ssl-connection) due to the use of a self-signed certificate for database connections by Supabase. |
-| Neon     | _Coming_ | Support details coming soon.                                                                                                 |
+| Neon     | _Coming_ | Support details coming soon.                                                                                                                                                     |
 | AWS RDS  | _Coming_ | Support details coming soon.                                                                                                                                                     |
+
+## Which regions is Prisma Pulse available in?
```

Comment:
Offtopic for this PR?

Reply:
@janpio, it is off-topic, but I included it as I'm referring to this in the CLI command to enable Prisma Pulse. 
Particularly for the `region` flag during creation.